src/Uno.UI.Composition/Composition/CompositionObject.SubPropertyHelpers.cs

```diff
@@ -0,0 +1,323 @@
+#nullable enable
+
+using System;
+using System.Numerics;
+using Windows.UI;
+
+namespace Microsoft.UI.Composition;
+
+partial class CompositionObject
+{
+	private protected Color GetColor(ReadOnlySpan<char> subPropertyName, Color existingValue, object? propertyValue)
```

Comment:
```suggestion
	private protected Color UpdateColor(ReadOnlySpan<char> subPropertyName, Color existingValue, object? propertyValue)
```

Not sure if it's the right name, but for me I would not expect a `Get<Something>` to update only sub properties. (Same for all methods below)

Reply:
Fixed.